extensions/project-descriptor.md

```diff
@@ -163,11 +163,18 @@ This table MAY contain an array of buildpacks. The schema for this table is:
 id = "<buildpack ID (optional)>"
 version = "<buildpack version (optional default=latest)>"
 uri = "<url or path to the buildpack (optional default=urn:buildpack:<id>)"
+
+[io.buildpacks.build.buildpacks.script]
+api = "<buildpack api>"
+shell = "<string (optional default=/bin/sh)>"
+inline = "<script contents (optional)>"
```

Comment:
This isn't really optional right? If the script table is defined inline is required.

Reply:
yea, i think that's a mistake

Reply:
Fixed in dc1b9ac